tests/cli/test_history.py

```diff
@@ -19,7 +19,7 @@ def test_build_history_list():
         terse=False, no_header=False, pager=False, color=Color.default().name
     )
 
-    # test with pager support: buildtest history list --pager
+    # test with pager support: buildtest history --pager list
```

Comment:
remove `test with pager support:` just have the command and make this `buildtest history list --pager`